app/assets/stylesheets/hyrax/_dashboard.scss

```diff
@@ -123,7 +123,7 @@ ul.collapsing > li > a {
   padding-left: 40px;
 }
 
-.collapse-toggle {
+li > .collapse-toggle {
```

Comment:
Can we use `.nav-stacked` (or add a new class to the admin sidebar) instead of `li >`. I think that makes it clearer that it's for the admin menu.